### DISCUSSION TITLE
How to convert multiple files with .safetensors extension to 1 gguf file so that jan can use it?

### DISCUSSION THREAD

**Discussion by msjsc001** [janhq/jan#2060] [Get Help] [answered]
Question for everyone, https://llm.extractum.io/list/ has a lot of models but jan can't use them directly at the moment. Until jan supports safetensors, how to convert multiple files with .safetensors extension to 1 gguf file so that jan can use them. (I don't know how to program but would like to use more models)

**Comment by louis-jan** [comment, marked as answer]
I'm not sure if this is still relevant, but you can follow this instruction to quantize SafeTensors models.
https://qwen.readthedocs.io/en/latest/quantization/llama.cpp.html
